### ISSUE TITLE
Stream audio and video out of sync n spite of entering a sync-offset value

### ISSUE THREAD

**Issue by Sync-offset** [obsproject/obs-studio#3663]
## Platform

Operating system and version: Windows 10
OBS Studio version: 26.0.2

## Expected Behavior

To sync Audio and Video I enter a sync offset of 500ms
![afbeelding](https://user-images.githubusercontent.com/73615551/97555700-b548a780-19d8-11eb-8e37-b3910d8ca41b.png)

Audio should be in Sync in the program view, the recording and the stream.

## Current Behavior

Program view is in Sync. Recording and stream are out of sync.
When I start the stream and than re-enter the offset value, the stream gets in sync.

## Steps to Reproduce

1. Enter an offset value
2. Start the stream
3. Check the stream: it is out off sync.
4. Re-enter the same offset value
5. Check the stream: it is in sync.

## Additional information

This problem started after updating from 25.0.8 to 26.0.0 and 26.0.2
Re-installing 25.0.8 solved the problem for now.

**Comment by WizardCM** [comment]
Sync offset is not applied to monitoring. As you are monitoring your microphone, and then capturing that using desktop audio, it will remain out of sync until you stop monitoring.

**Comment by Sync-offset** [comment]
The problem is not in the monitoring signal, but in the recording and the stream signal.

**Comment by WizardCM** [comment]
Monitor to an audio output device that isnt being captured.

**Comment by Sync-offset** [comment]
![afbeelding](https://user-images.githubusercontent.com/73615551/97557743-7bc56b80-19db-11eb-9b5d-2d58ed8890ef.png)

Monitor and output in ONE selection-option.
With the exact same settings this problem did not excist in version 25.0.8

**Comment by taradinoc** [comment]
I can confirm audio sync offset isn't working as expected in OBS 26.0.2. If I set a sync offset of 7000 ms on a media source (MP3), the audio isn't delayed on the monitoring output _or_ the recording.

**Comment by LuminairPrime** [comment]
It's 2 years later, but I just tested "sync offset", and it seems to work now. Please note that entering a negative offset doesn't seem to be a thing, so don't put a - in there. This feature seems to be undocumented, but it just delays the audio into the future.

Here are some videos to help you test:
https://www.youtube.com/watch?v=ucZl6vQ_8Uo
https://www.youtube.com/watch?v=TjAa0wOe5k4

**Comment by Fenrirthviti** [comment]
Hi, this seems to have been corrected since this was originally posted, is this still an issue on current (28.0.3) for you @Sync-offset?

**Comment by Sync-offset** [comment]
Hi Joel,

This problem is solved.

I don’t know if its oke in 28.0.3.
I did not update to the latest version because NDI plugins are not updated for that.

Thank for you response.

René



Van: Joel Bethke
Verzonden: woensdag 19 oktober 2022 22:04
Aan: obsproject/obs-studio
CC: Sync-offset; Mention
Onderwerp: Re: [obsproject/obs-studio] Stream audio and video out of sync nspite of entering a sync-offset value (#3663)

Hi, this seems to have been corrected since this was originally posted, is this still an issue on current (28.0.3) for you @Sync-offset?
